Source/WebCore/dom/Element.cpp

```diff
@@ -3894,14 +3894,14 @@ String Element::title() const
     return String();
 }
 
-const AtomString& Element::pseudo() const
+const AtomString& Element::userAgentPart() const
 {
-    return attributeWithoutSynchronization(pseudoAttr);
+    return attributeWithoutSynchronization(useragentpartAttr);
 }
```

Comment:
Probably a follow-up, but this should return nothing outside of UA shadow trees (or better, assert that it's not called outside)